client/src/glossary/glossary.yaml

```diff
@@ -12,4 +12,36 @@ glossary_translation:
 
 glossary_related_data:
   en: 'Related Data:'
-  fr: 'Données connexes :'
\ No newline at end of file
+  fr: 'Données connexes :'
+
+glossary_placeholder: 
+  en: "Search for a term"
```

Comment:
Outer quotations not necessary, or standard for the project, in text maker yaml. I'm surprised to see that they get stripped out somewhere and aren't appearing in the app!